styles/_hero.scss

```
@@ -1,7 +1,9 @@
 /***** Hero component *****/
 
 .hero {
-  background: #2a2929 url($mit_background) repeat-x 50% 100%;
+  display: flex;
+  justify-content: center;
+  background: #2a2929 url($ocw_background) repeat-x 50% 100%;
```

Don't think we need this background color. Only image would be fine
```suggestion
  background: url($ocw_background) repeat-x 50% 100%;
```